src/Sylius/Component/Core/Model/AvatarImage.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Model;
+
+class AvatarImage extends Image implements AvatarImageInterface
```

Comment:
Why did you use AvatarImage instead just Image?

Reply:
Because Image is abstract class